_data/pt-BR.yml

```
@@ -0,0 +1,294 @@
+  currentVersion: Versão Atual
+  hFeatures: Funcionalidades
+  hIdentityVerifications: Sem Verificação de Identidade
+  pIdentityVerifications: Instantaneamente acessível. Não há necessidade de registro ou aprovação de uma autoridade central.
+  hDecentralized: Descentralizado
+  pDecentralized: A negociação acontece numa rede P2P global de usuários rodando Bisq nas suas próprias máquinas.
+  hSafe: Seguro
+  pSafe: "Bisq nunca custodia seus fundos. Depósitos de segurança e carteiras para negociação multi-assinatura promovem negociações seguras e bem-sucedidas."
+  hPrivate: privado
+  pPrivate: "Ninguém, exceto parceiros de negociação, troca dados de identificação pessoal. Todos os dados pessoais são armazenados localmente."
+  hSecure: Seguro e sempre funcionando
+  pSecure: Comunicação criptografada de ponta a ponta roteada através do Tor e sem tempo de inatividade.
+  hOpen: Aberto
+  pOpen: "<p class=\"grey px-md-5\">O código é <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">aberto</a>, e a estratégia do projeto é discutida abertamente no <a href=\"https://bisq.network/slack-invite\" target=\"_blank\" rel=\"noopener\">Slack</a> e <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">GitHub</a>.</p>"
+  hEasy: Fácil de Usar
+  pEasy: "<p class=\"grey px-md-5\">Nós fizemos toda essa sofisticação simples—<a href='https://docs.bisq.network/getting-started' target='_blank' rel=\"noopener\">faça sua primeira negociação</a> em menos de 10 minutos!</p>"
+  hBisqDao: A DAO Bisq
+  pBisqDao1: Governança descentralizada para uma exchange descentralizada.
+  pBisqDao2: Saiba mais aqui
+  hGetStarted: Como começar
+  divGetStarted: "Com o Bisq, você pode começar a fazer negociações em menos de 10 minutos:"
+  liGetStarted1: Baixe o app
+  liGetStarted2: ", ou compile do código fonte."
+  liGetStarted3: >
+    <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Adicione suas contas de moeda nacional ou cripto.</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Crie uma nova oferta ou execute uma já existente.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Siga a interface para finalizar a negociação.</li>
+  hJoin: Junte-se à Comunidade
+  pJoin: >
+    <p>Bisq está sempre à procura de desenvolvedores, designers e especialistas em segurança altamente qualificados para se juntarem à nossa <a href="https://bisq.network/community/">comunidade</a>. Se você tem interesse em  melhorar a Bisq, há muitas maneiras de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>. Gostaríamos muito de <a href="https://bisq.network/community/">ouvir você</a>!</p> <p>O nosso <a href="https://bisq.network/roadmap/">roadmap</a> oferece uma visão geral dos recursos e marcos de alto nível no caminho para a Bisq v1.0.</p>
```

```suggestion
    <p>Bisq está sempre à procura de desenvolvedores, designers e especialistas em segurança altamente qualificados para se juntarem à nossa <a href="https://bisq.network/community/">comunidade</a>. Se você tem interesse em  melhorar a Bisq, há muitas maneiras de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>. Gostaríamos muito de ouvir você!</p> <p>O nosso <a href="https://bisq.network/roadmap/">roadmap</a> oferece uma visão geral dos recursos e marcos de alto nível no caminho para a Bisq v1.0.</p>
```

Removed extra link to community page, to keep consistent with changes from #287.